integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts

```diff
@@ -89,11 +90,9 @@ medusaIntegrationTestRunner({
       beforeEach(async () => {
         await createAdminUser(dbConnection, adminHeaders, appContainer)
 
-        // Here, so we don't have to create a region for each test
-        defaultRegion = await regionModuleService.create({
-          name: "Default Region",
-          currency_code: "dkk",
-        })
+        const { region } = await seedStorefrontDefaults(appContainer, "dkk")
```

Comment:
@adrien2p @sradevski keeping this in the test file as its not required for most specs.

Reply:
sounds good 👍